FeatureSuggestion/ShaktiPrasad_SystemEntities_Suggestions.md

```diff
@@ -0,0 +1,44 @@
+#### This may not cover all required entities. Just putting forward my thoughts.
+
+#### Class Section
+{
+````
+	string title;
+	string paragraph;(rich text)
+	Public:
+		Section(title, paragraph)
+		void setTitle();
+		void getTitle();
+		void setParagraph();
+		void getParagraph();
+````
+}
+
+#### Class Resume
+{
+````
+	/* This is a base class,Each template shall inherit this, additonal sections vars can be added to each template. */
+
+	/*
+	Each template can have 
+	some fixed sections.
+	*/
+	Section personalInfo;
```

Comment:
We shouldn't fix any of the sections. It should be a map and every section should be removable.